Editing Covid_Tracker README File

chrystal00 opened the pull request:
This is part of an assignment for my CIS4930 course where we have to edit a README file in order to help us with proper documentation.

chrystal00 committed:
Added Description Title

chrystal00 committed:
Added Description Title

chrystal00 committed:
Condensed Description

chrystal00 committed:
Condensed API Link

chrystal00 committed:
Got Rid of Logos

arafaysaleem commented on README.md:
```diff
@@ -35,7 +36,7 @@ The UI is inspired by a design on [Dribbble](https://dribbble.com/shots/10847147
 
 *Feel free to fork and contribute to include these features.* ❤︎
 
-## 🚀 Technologies
```
I want to keep the logos

chrystal00 replied:
This is just for a school assignment where I make suggestions to a README file so you can take me suggestions with a grain of salt.

arafaysaleem reviewed (changes requested):
I need to keep the logos for appeal.

chrystal00 committed:
Updated Features

chrystal00 committed:
Added Prereq & Installation 

Got rid of "Technologies" and sub header of "How to start with Flutter and Dart" just to categorize everything and make it simpler  to read.

chrystal00 committed:
Added Table of Contents

chrystal00 committed:
Formatting Changes

chrystal00 committed:
Formatting Changes

chrystal00 committed:
Added Debugging

chrystal00 committed:
Update README.md
